relay-server/src/services/buffer/mod.rs

```diff
@@ -119,6 +160,7 @@ pub struct Services {
 /// Spool V2 service which buffers envelopes and forwards them to the project cache when a project
 /// becomes ready.
 pub struct EnvelopeBufferService {
+    partition_id: u32,
```

Comment:
We should add the `partition_id` to all the statsd metrics.

Reply:
Will do, this is critical.